python/paddle/distributed/auto_parallel/parallelizer.py

```diff
@@ -185,10 +188,9 @@ def _get_dist_program(self, rank, dist_context=None, relaunch_phase=False):
             self._parameter_list, self._no_grad_set, self._callbacks)
 
         # serial forward pass
-        self._apply_serial_pass(completed_main_program, serial_startup_program)
-
+        self._apply_serial_pass(completed_main_program, serial_startup_program,
```

Comment:
Please change `_apply_serial_pass()` to `_apply_pre_passes()`.

Reply:
fixed~